includes/fs-essential-functions.php

```diff
@@ -36,6 +36,39 @@ function fs_normalize_path( $path ) {
 		}
 	}
 
+	if ( ! function_exists( 'fs_get_plugins' ) ) {
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since 2.2.1
+         *
+         * @param string $plugin_folder
+         * @param bool   $delete_cache
+         *
+         * @return array
+         */
+	    function fs_get_plugins( $plugin_folder = '', $delete_cache = false ) {
+            if ( ! $cached_plugins = wp_cache_get( 'plugins', 'plugins' ) ) {

```

Comment:
@fajardoleo this could be quite confusing together with the `!`. I would prefer if you move the assignment outside the `if`.